src/test/java/org/broadinstitute/consent/http/AbstractTest.java

```diff
@@ -0,0 +1,24 @@
+package org.broadinstitute.consent.http;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.apache.commons.lang3.RandomUtils;
+
+public abstract class AbstractTest {
```

Comment:
I find this name a little confusing. It's not really a test, maybe test helper?

Reply:
It is essentially a test helper - I can rename it 👍🏽